agent/engine/docker_client_test.go

```diff
@@ -479,21 +468,17 @@ func TestStopContainerTimeout(t *testing.T) {
 	defer done()
 
 	warp := make(chan time.Time)
-	wait := &sync.WaitGroup{}
-	wait.Add(1)
 	mockDocker.EXPECT().StopContainerWithContext("id", uint(client.config.DockerStopTimeout/time.Second), gomock.Any()).Do(func(x, y, z interface{}) {
```

Comment:
I think this needs `.MaxTimes(1)` as well. So do all other `EXPECT()`, where we are actually letting these timeouts time out (wherever `xContainerShortTimeout` is used).

Reply:
OK, added `.MaxTimes(1)` to all timeout tests in docker client.